samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/ChatHistoryController.cs

```diff
@@ -35,42 +36,48 @@ public class ChatHistoryController : ControllerBase
     /// <param name="logger">The logger.</param>
     /// <param name="chatSessionRepository">The chat session repository.</param>
     /// <param name="chatMessageRepository">The chat message repository.</param>
+    /// <param name="chatParticipantRepository">The chat participant repository.</param>
     /// <param name="promptsOptions">The prompts options.</param>
     public ChatHistoryController(
         ILogger<ChatHistoryController> logger,
         ChatSessionRepository chatSessionRepository,
         ChatMessageRepository chatMessageRepository,
+        ChatParticipantRepository chatParticipantRepository,
         IOptions<PromptsOptions> promptsOptions)
     {
         this._logger = logger;
         this._chatSessionRepository = chatSessionRepository;
         this._chatMessageRepository = chatMessageRepository;
+        this._chatParticipantRepository = chatParticipantRepository;
         this._promptOptions = promptsOptions.Value;
     }
 
     /// <summary>
     /// Create a new chat session and populate the session with the initial bot message.
+    /// The regex pattern that is used to match the user id will match the following format:
+    ///    - 2 period separated groups of one or more hyphen-delimited alphanumeric strings.
+    /// The pattern matches two GUIDs in canonical textual representation separated by a period.
```

Comment:
Technically this matches any dash-delimited number of characters. Maybe add an example: "a1-2b-3c.x4-5y-6z"
```suggestion
    /// The pattern in meant to match two GUID-like strings in canonical textual representation separated by a period (e.g., a1-2b-3c.x4-5y-6z)
```